src/components/dialogs/network-modifications/vsc/hvdc-line-pane/vsc-hvdc-line-pane-utils.tsx

```diff
@@ -34,6 +34,11 @@ export function getVscHvdcLinePaneSchema(id: string) {
                 [P0]: yup
                     .number()
                     .nullable()
+                    .when([ANGLE_DROOP_ACTIVE_POWER_CONTROL], {
+                        is: (angleDroopActivePowerControl: boolean) => angleDroopActivePowerControl,
```

Comment:
direct `form is: true,`  in this case
you don't need to use unnecessary function to return a boolean value that is already known